ci-operator/step-registry/redhat-chaos/application-outages/console/observer/redhat-chaos-application-outages-console-observer-commands.sh

```
@@ -43,5 +43,9 @@ rc=$?
 if [[ $TELEMETRY_EVENTS_BACKUP == "True" ]]; then
     cp /tmp/events.json ${ARTIFACT_DIR}/events.json
 fi
+
+if [[ $CAPTURE_METRICS == "True" ]]; then 
+  cp /home/krkn/kraken/config/metrics-report.yaml ${ARTIFACT_DIR}/metrics-report.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)